.changeset/olive-yaks-repeat.md

```diff
@@ -0,0 +1,6 @@
+---
+"@logto/core": minor
+"@logto/integration-tests": minor
+---
+
+update webhook API
```

Comment:
nit: I think we should emphasis that there's a breaking change

Reply:
updated.